eng/testing/performance/performance-setup.sh

```diff
@@ -191,7 +191,7 @@ payload_directory=$source_directory/Payload
 performance_directory=$payload_directory/performance
 benchmark_directory=$payload_directory/BenchmarkDotNet
 workitem_directory=$source_directory/workitem
-extra_benchmark_dotnet_arguments="--iterationCount 1 --warmupCount 0 --invocationCount 1 --unrollFactor 1 --strategy ColdStart --stopOnFirstError true"
+extra_benchmark_dotnet_arguments="--iterationCount 1 --warmupCount 0 --invocationCount 1 --unrollFactor 1 --strategy ColdStart --stopOnFirstError true --logBuildOutput"
```

Comment:
Looks like this can get reset for `$internal == true` on line 204. Maybe we can add the argument at the end, like after line 309?